pkg/server/status/runtime.go

```diff
@@ -419,6 +427,13 @@ func (rsr *RuntimeStatSampler) SampleEnvironment(ctx context.Context) {
 	goAllocated := ms.Alloc
 	goTotal := ms.Sys - ms.HeapReleased
 
+	totalMemory, err := GetTotalMemory(ctx)
+	if err != nil {
+		log.Warningf(ctx, "problem fetching total memory available to process: %s; memory percent metric will be empty.", err)
```

Comment:
I would skip this warning, we already have it at node startup, no reason to log it again every 10 seconds. Definitely skip setting the metric though.